### PR TITLE
CRUD recipes

### PR DESCRIPTION
**Pull request by sdymj84**
- A lot of files are changed by Prettier (single quote to double quotes and add semicolons)
- There's a lot of redundant commits because I played with git command just for tests and practice. I will squash when merging

- main changes are recipes update and delete routes, test, doc

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -1,19 +1,17 @@
-const express = require('express');
-const cors = require('cors');
-require('dotenv').config();
+const express = require("express");
```
Comment (sdymj84) [inline backend/app.js]: prettier changes - nothing to review

File: backend/routes/users.js [hunk 2]
```diff
@@ -1,39 +1,46 @@
-const router = require('express').Router();
-const User = require('../models/user.model');
-const firebase = require('../util/fbConfig')
-const { validateSignupData, validateSigninData } = require('../util/validators')
-const fbAdmin = require('../util/fbAdminConfig')
```
Comment (kststudio) [inline backend/routes/users.js:5]: This isn't being used at all in this document. Should we remove it or is it needed for something that hasn't been built yet?

File: backend/tests/users/signin.test.js [hunk 3]
```diff
@@ -8,106 +8,85 @@
   6. should throw error when password is wrong
 */
 
-const { setupDB, request, fbAdmin } = require('../test-setup')
-setupDB('signin-test')
+const { setupDB, request, fbAdmin } = require("../test-setup");
```
Comment (kststudio) [inline backend/tests/users/signin.test.js]: fbAdmin is bot being used here.

File: backend/tests/users/signin.test.js [hunk 4]
```diff
@@ -8,106 +8,85 @@
...
+const { getToken, signup, deleteUser } = require("../auth-setup");
```
Comment (kststudio) [inline backend/tests/users/signin.test.js]: getToken is not being used here

File: backend/tests/users/update.test.js [hunk 5]
```diff
@@ -5,86 +5,53 @@
...
-  //   expect(res.body.error.code).toBe("auth/empty-email")
-  //   done()
-  // })
-
-
-
-
-
-
+const { setupDB, request, fbAdmin, fb } = require("../test-setup");
```
Comment (kststudio) [inline backend/tests/users/update.test.js]: fbAdmin and fb are not being used here

**Review by kststudio — CHANGES_REQUESTED** [review]
This looks good. Looks like a lot was cleaned up. Check out some of my comments on a few files.

File: backend/routes/users.js (hunk 2 above)
Comment (sdymj84) [reply backend/routes/users.js:5]: oh yes, I'm deleting it

File: backend/tests/users/signin.test.js (hunk 3 above)
Comment (sdymj84) [reply backend/tests/users/signin.test.js]: deleted

File: backend/tests/users/signin.test.js (hunk 4 above)
Comment (sdymj84) [reply backend/tests/users/signin.test.js]: deleted

File: backend/tests/users/update.test.js (hunk 5 above)
Comment (sdymj84) [reply backend/tests/users/update.test.js]: deleted

**Comment by sdymj84** [comment]
I removed variables not being used as you commented. please review those changes
Thank you!

**Review by kststudio — APPROVED** [review]

**Pull request merged**